src/net/conn_plain.c

```diff
@@ -25,6 +27,24 @@ set_error(int err)
 #endif
 }
 
+static int
+get_error(void)
+{
+#ifdef WIN32
+	return WSAGetLastError();
+#else
+	return errno;
+#endif
+}
+
+/* We cannot define `pg_strerror` here because there is a #define in PG12 that
```

Comment:
The markdown formatting seems unnecessary in C comments

Reply:
I use this to avoid confusing when mixing English text with references to symbols in the code.